pipeline-ui/src/components/WalletConnect.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'react';
+const { AlgoAddress, Pipeline, AlgoSignerButton } = require('pipeline-ui');
+
+interface WalletConnectProps {
+	updateAddress: Function;
+}
+
+interface WalletConnectState {
+	Algaddress: string;
```

Comment:
what's `Algaddress`?

Reply:
Pipeline documentation had suggested to use this name. I have changed it to just `address`

Reply:
thanks